File: airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/NoopV2TableMigrator.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.base.destination.typing_deduping;

public class NoopV2TableMigrator implements V2TableMigrator {

  @Override
  public void migrateIfNecessary(final StreamConfig streamConfig) {
    // do nothing
  }

}

File: airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/NoopV2TableMigrator.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.integrations.base.destination.typing_deduping

class NoopV2TableMigrator : V2TableMigrator {
    override fun migrateIfNecessary(streamConfig: StreamConfig?) {
        // do nothing
    }
}
